examples/sample-app/README.md

```diff
@@ -3,7 +3,11 @@ OpenShift 3 Application Lifecycle Sample
 
 This is a set of configuration files and scripts which work with OpenShift 3 to create a new application and perform application builds.
 
-This example assumes you have successfully built the `openshift` binary executable and have Docker installed/working.  See https://github.com/openshift/origin/blob/master/CONTRIBUTING.adoc.
+This example assumes you have successfully built the `openshift`
+binary executable, and have Docker installed/working.  See
+https://github.com/openshift/origin/blob/master/CONTRIBUTING.adoc. `osc`
+should be either a symlink or a copy of `openshift` (it serves as a
+shortcut for `openshift cli`).
```

Comment:
sure, but it doesn't necessarily put openshift in their path either, so couldn't you just say "assumes you have built openshift and have the osc symlink in your path"?